dev_envs/mac-env-setup.md

```diff
@@ -0,0 +1,260 @@
+
+1. Fill your `name` and `email`, using the same values as above
+1. Copy the alphanumeric string from the key generation output line that says
+    `gpg: key EXAMPLE1234ABCDE marked as ultimately trusted`
+1. Fill the alphanumeric signing key string in the `[user]` section:
+    `signingkey = EXAMPLE1234ABCDE`
+
+Next, export the public key via command line and add it to your GitHub account:
+
+1. `gpg --armor --export user@example.com`
+1. Copy the output, starting from `-----BEGIN PGP PUBLIC KEY BLOCK-----`
+1. Add a new PGP key to your [GitHub keys page](https://github.com/settings/keys)
+1. Paste the public key into the dialog and `Add GPG key`
+1. Confirm your password to continue (if requested)
+
+Lastly, enable the global configuration option for commit signing:
+
+```bash
+git config --global commit.gpgsign true
+```
+
+#### Generate `ssh` key to use git on the command line ####
+
+Reference: [Github: Generating a new SSH key](https://docs.github.com/en/github/authenticating-to-github/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent#adding-your-ssh-key-to-the-ssh-agent)
+
+Note: use `ed25519` instead of `rsa`
+
+```bash
+ssh-keygen -t ed25519
+```
+
+1. Save the generated key in the default location (e.g. `~/.ssh/id_ed25519`)
+1. Use a passphrase you'll remember and/or save in your keychain
+1. Start the `ssh-agent` in the background and add your new credential:
+
+    ```bash
+    eval "$(ssh-agent -s)"
+    ssh-add -K ~/.ssh/id_ed25519
+    ```
+
+#### Add ssh key to GitHub ####
+
+Reference: [Github: Adding a new SSH key to your GitHub account](https://docs.github.com/en/github/authenticating-to-github/adding-a-new-ssh-key-to-your-github-account)
+
+Note: use `ed25519` instead of `rsa`
+
+```bash
+$ pbcopy < ~/.ssh/id_ed25519.pub
+# Copies the contents of the id_ed25519.pub file to your clipboard
+```
+
+1. Load your [Github Settings page for adding an SSH key](https://github.com/settings/ssh/new)
+and paste the contents of your clipboard
+1. Give this device a descriptive name and `Add SSH key`
+1. Confirm your password to continue (if requested)
+
+## Setup for the COOL ##
+
+There are some additional setup steps to access the
+[COOL (Cloud-Optimized Operations Lab)](https://github.com/cisagov/cool-system/wiki/):
+
+1. [Install MIT Kerberos](https://github.com/cisagov/cool-system/wiki/Installing-MIT-Kerberos)
+    - NOTE: If you chose Automatic Installation above, this step was already
+    performed by the `laptop` script
+1. [Configuring Kerberos for the COOL](https://github.com/cisagov/cool-system/wiki/Configuring-Kerberos-for-the-COOL)
+1. [Configuring your browser for the COOL](https://github.com/cisagov/cool-system/wiki/Configuring-your-browser-for-the-COOL)
+
+After you have successfully completed the tasks above, follow
+[these instructions to access Guacamole](https://github.com/cisagov/cool-system/wiki/Accessing-an-assessment-environment-with-Guacamole).
```

Comment:
We may want to clarify this slightly as:
```
[these instructions to access Guacamole](https://github.com/cisagov/cool-system/wiki/Accessing-an-assessment-environment-with-Guacamole).
Note: You will need to have been given access to one or more COOL environments
in order to access anything with Guacamole.
```

Reply:
I suggest "Note: " :arrow_right:  "Note that "

Reply:
Resolved in c1026d4